src/views/Transfer/Transfer.js

```diff
@@ -111,7 +119,8 @@ export default {
                     this.selectedAccount.keyPair,
                     this.formObject.to.address,
                     new Number(this.formObject.amount),
-                    this.formObject.message
+                    this.formObject.message,
+                    this.selectedCurrency
                 );
 
```

Comment:
Catch should be split up between buildFundedPaymentTransaction and submitFundedTransaction

Reply:
hasnt been changed in this pr?